rules/using-digital-asset-manager/rule.md

```diff
@@ -0,0 +1,98 @@
+6. Finally export each clip individually to your stock library.
+
+From there you can use a digital asset manager like Adobe Bridge to add further metadata information to help with navigating your personal stock footage library.
+
+## What is Adobe Bridge?
+
+Adobe Bridge is a creative digital asset manager that lets you preview, organize, edit, and publish multiple creative assets (including Adobe Photoshop, InDesign, Illustrator, After Effects, and Dimension files) with thumbnails and rich previews. Edit metadata. Add keywords, labels, and ratings to assets. Organize assets using collections, and find assets using powerful filters and advanced metadata search features.
+
+## Organize your media files
+
+By creating a well-structured folder hierarchy and sticking to a standard naming convention, we can make sure media assets won't get accidentally deleted or go missing.
+Consider organizing videos based on projects, clients, or types of footage (e.g., interviews, B-roll, aerial shots). 
+
+Learn more on:
+
+* [Video data - Do you back up your files in an organized way?](https://www.ssw.com.au/rules/organize-and-back-up-your-files/)
+
+## Utilize metadata
+
+Leverage the power of metadata with tags and or keywords and enhance searchability. In Adobe Bridge you can provide additional information about your media asset. Add relevant details such as project name, shoot location, keywords, and descriptions to your file, doing so will make it significantly easier to search for specific videos and streamline your workflow.
+
+![Figure: Adobe Bridge metadata tab](scr-20230809-odug.png "Figure: Adobe Bridge metadata tab")
+
+![Figure: Adobe Bridge keyword list tab](scr-20230809-oeed.png "Figure: Adobe Bridge keyword list tab")
+
+## Importing and Exporting Keywords for team collaboration
+
+Although Adobe Bridge is made without multi-user support. With a workaround, It is still possible to use this program in a small team, this is made possible due to the ability for keywords to be written onto each file directly. So as long as your team uses a shared keywords list, searching your media library should be simple.
+You can do this by importing and exporting keywords .XML file for your team to use. 
+
+1. First Navigate to the Keywords tab and right-click the tab to open the dropdown menu.
+
+![Figure: Adobe Bridge Keywords tab location](scr-20230809-nqcl.png "Figure: Adobe Bridge Keywords tab location")
+
+2. Next, click on export to export your current keyword list.
+
+![Figure: Export keywords list](export-scr.png "Figure: Export keywords list")
+
+3. Save this file with a version number to a shared folder accessible to everyone in your team.
+4. To import an existing keyword list, navigate and click on import from the same list. Then select the .XML file you want to import.
+
+![Figure: Import keywords list](import-scr.png "Figure: Import keywords list")
+
+Know more on:
```

Comment:
You already have these 2 rules on the Related list... so this last section is not necessary...

Reply:
Done